go/internal/sidecar/hotpath_relay_test.go

```
@@ -0,0 +1,755 @@
+		extra map[string]any
+		want  int
+	}{
+		{"materially armed item ID repair", map[string]any{"responsesItemIdRepair": map[string]any{"repairInvalidIds": true}}, http.StatusOK},
+		{"empty item ID repair remains relay-safe", map[string]any{"responsesItemIdRepair": map[string]any{"message": []any{}}}, http.StatusOK},
+		{"snapshot repair", map[string]any{"responsesSnapshotRepair": true}, http.StatusOK},
+		{"stateless Responses", map[string]any{"statelessResponses": true}, http.StatusServiceUnavailable},
+		{"preserved reasoning model is case-insensitive", map[string]any{"preserveReasoningContentModels": []any{"TEST-MODEL"}}, http.StatusServiceUnavailable},
+		{"different preserved reasoning model remains relay-safe", map[string]any{"preserveReasoningContentModels": []any{"other-model"}}, http.StatusOK},
+	}
+	for _, c := range cases {
+		c := c
```

golangci-lint (2.13.2)

[error] The copy of the 'for' variable "c" can be deleted (Go 1.22+)

(copyloopvar)